internal/wordvault/server.go

```diff
@@ -32,7 +32,7 @@ var ErrNeedMembership = errors.New("adding these cards would put you over your l
 var ErrMaybeRefreshApp = invalidArgError("Card with your input parameters was not found. Please refresh this page as the app may have updated.")
 var ErrMaintenance = connect.NewError(connect.CodeUnavailable, errors.New("WordVault App is currently undergoing maintenance. Please wait a few moments and try again."))
 
-const JustReviewedInterval = time.Second * 10
+const JustReviewedInterval = time.Second * 5
```

Comment:
Was hitting this error when testing things locally, with short-term scheduling a 10-second review seems soooortttaa plausible if someone has cleared their deck and just added some new cards (which they keep missing) so figured it may be nice to ratchet this down. No strong opinion on it though

Reply:
@BenMusch It seems the shortest interval that can be is 1 minute:

https://github.com/open-spaced-repetition/go-fsrs/blob/main/scheduler_basic.go#L33

I would actually prefer to make it a little bit _longer_ (30 seconds maybe). The reason is when studying with bad internet sometimes it can take 10 seconds or more for it to give you back an answer, and if you click again then it'll re-grade it.

Reply:
Let me try to repro -- my cardbox was quite large so there's a chance it was actually just that I was clicking fast enough to double-send a review for the same card (e.g. the exact error this was actually trying to catch)